nncf/openvino/graph/layer_attributes.py

```diff
@@ -67,9 +67,100 @@ def get_const_port_ids(self) -> List[int]:
             return list(self._constant_attributes.keys())
         return []
 
+    def get_backend_agnostic_attributes(self):
+        return self._layer_attributes
+
+
+OV_CONV_METATYPES = [
+    OVConvolutionMetatype,
+    OVDepthwiseConvolutionMetatype,
+    OVGroupConvolutionMetatype,
+    OVConvolutionBackpropDataMetatype,
+    OVGroupConvolutionBackpropDataMetatype,
+]
+
+
+OVConvLayout = List[OVLayoutElem]
+
+
+def get_conv_weights_layout_from_node(node: NNCFNode) -> OVConvLayout:
+    """
+    Calculates weights layout for a target convolution node.
+
+    :param node: Target convolution node.
+    :return: Target convolution Node weights layout.
+    """
+    layer_attributes = node.layer_attributes
+    port_id = _get_constant_port_id_from_layer_attributes(layer_attributes)
+    return get_conv_weights_layout(
+        ov_metatype=node.metatype, weights_shape=layer_attributes.constant_attributes[port_id]["shape"]
+    )
+
+
+def get_linear_weights_layout_from_node(node: NNCFNode) -> OVConvLayout:
```

Comment:
Again, why are you returning `OVConvLayout` for linear ops? For me `List[OVLayoutElem]` is less confusing

Reply:
That is fair, I've fixed it, thanks